lib/live_debugger/services/callback_tracer/actions/state.ex

```diff
@@ -18,6 +19,11 @@ defmodule LiveDebugger.Services.CallbackTracer.Actions.State do
     do_save_state!(pid)
   end
 
+  def maybe_save_state!(%Trace{pid: pid, function: function, args: [_, _, socket]})
+      when function in [:mount, :handle_params] do
+    do_save_initial_state!(pid, socket)
```

Comment:
I assume that this variant is used for LiveViews that dies immediately and we cannot fetch the state as usual via `LiveViewDebug.liveview_state/1` because it may be already dead, right? The problem with this approach is that you don't have a list of LiveComponents in the state storage which theoretically may affect LiveViews that do not die immediately.

What will happen if someone triggers `handle_params/3` that does not change any assigns? `handle_params/3` in this case will be the last callback triggered in given LiveView (`render/1` will not be triggered because assigns did not change). In such case components inspection may break